test/models/transcode_command_test.rb

```diff
@@ -63,13 +63,15 @@ class TranscodeCommandTest < ActiveSupport::TestCase
       track_title: 'track-title',
       track_number: 'track-number',
       album_title: 'album-title',
-      artist_name: 'artist-name'
+      artist_name: 'artist-name',
+      release_date: 'release-date'
     }
     command_string = TranscodeCommand.new(@input, @output, :flac, metadata).generate
     assert_contains_pair(command_string, ['-metadata', 'TITLE="track-title"'])
     assert_contains_pair(command_string, ['-metadata', 'TRACKNUMBER="track-number"'])
     assert_contains_pair(command_string, ['-metadata', 'ALBUM="album-title"'])
     assert_contains_pair(command_string, ['-metadata', 'ARTIST="artist-name"'])
+    assert_contains_pair(command_string, ['-metadata', 'RELEASEDATE="release-date"'])
```

Comment:
See above comment regarding field naming.

Reply:
This has been updated to use `DATE` in the [follow-up commit](https://github.com/freerange/jam-coop/pull/222/commits/7a4242d7f4c3e5ce5c23a8b956003898ad0b9890)